<!DOCTYPE html>
<html>
	<link rel="stylesheet" type="text/css" href="index.css">
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
	<link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
		<title> Meije Mathé </title>
	</head>
	<body>
		<center>
			<div class="center">
				<a href = "index.html" style = "text-decoration:none">
					<p class="name"> M E I J E &emsp; M A T H É </p>
				</a>
			  		<ul class="menu">
						<li>
							 <a href="education.html"> EDUCATION </a>
						</li>
						<li>
							 <a href="experience.html"> EXPERIENCE </a>
						</li>
						<li class = "selected">
							<a href="projects.html"> PROJECTS </a>
						</li>
						<li>
							<a href="skills.html"> SKILLS </a>
						</li>
						<li>
							<a href="contact.html"> CONTACT </a>
						</li>
					</ul>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	Building a personal website from scratch
				    	<div class="card__details">
				 			<span class="year">hobby</span> <br>
				 			<span class="tag"> HTML </span>
				 			<span class="tag"> Javascript</span>
				 			<span class="tag"> CSS </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	The aim of this personal project is to apply and improve skills I learned in my Web Technologies course and create a cool extension of my resume that uniquely identifies me. <br> <br>
				    	I also took it as an opportunity to improve myself and work on self-motivation, attention to details and creatvity by designing everything from scratch.
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	Green BIM: Environmental impact of Bioinformatics
				    	<div class="card__details">
				 			<span class="year"> M2 </span> <br>
				 			<span class="tag"> Green Algorithms </span>
				 			<span class="tag"> Environment </span>
				 			<span class="tag"> Sustainability </span>
				 			<span class="tag"> Carbon Footprint </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	Here we present the Bioinformatics Master students’ <b>carbon consumption</b> during their apprenticeship. <br>
				    	We gathered all jobs carried out by the students for a month and determined their environmental impact using the <b>Green Algorithms</b> tool. <br>
				    	We studied how these results break down through the bioinformatics domains under scrutiny. <br> <br>
				    	<span class="link"> <i class="fa fa-file-pdf-o" aria-hidden="true"></i> <a href="./docs/poster_GreenBIM.pdf" target="_blank"> GreenBIM Poster </a> </span>
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	 Python programming for data analysis in mass spectrometry
				    	 <div class="card__details">
				 			<span class="year"> M2 </span> <br>
				 			<span class="tag"> Proteomics </span>
				 			<span class="tag"> Mass spectrometry </span>
				 			<span class="tag"> Python </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Comparison of <i>in silico</i> spectrums with a reference MGF file. <br> <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Set an ion intensity treshold for a set of spectrum and count the nunber of hits. Hits above the treshold will have a doubled score. <br> <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Add an acetylation mass to all candidate peptids and compute ions masses. <br> <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Return for each tested candidate peptid the number of hits, ranked by quality.
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	XSLT transformation project
				    	<div class="card__details">
				 			<span class="year"> M2 </span> <br>
				 			<span class="tag"> XSLT </span>
				 			<span class="tag"> XSL-FO </span>
				 			<span class="tag"> XML </span>
				 			<span class="tag"> HTML </span>
				 			<span class="tag"> CSS </span>
				 			<span class="tag"> Javascript </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	<b> Subject: </b> Create a HTML page and a PDF report from a XML file. <br><br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Write a <b>XSLT</b> transformation file to create an HTML page, according to a template, from a XSL data file. <br> <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Write a <b>XSL-FO</b> transformation file to create a PDF report from the same data file.
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	Multimodal representation learning from single-cell RNA-seq data
				    	<div class="card__details">
				 			<span class="year"> M2 </span> <br>
				 			<span class="tag"> Data integration </span>
				 			<span class="tag"> Single cell </span>
				 			<span class="tag"> RNA-seq </span>
				 			<span class="tag"> Kaggle </span>
				 			<span class="tag"> Python </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	<b> Subject: </b> Implementation and modification of the <b>LSL-LAE</b> (Living Systems Lab Auto Encoder) model. <br><br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Retrieve the LSL-AE code and run it on a <b>single-cell RNA-seq dataset</b>. <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Use code from the <b>JAE</b> (Joint Auto Encoder) to integrate <b>SVD (Singular Value Decomposition)</b> in the LSL-AE model. <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> <b>Benchmarking</b> of the AE and VAE (Variational Auto Encoder) models.
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	Epigenomics data analysis project
				    	<div class="card__details">
				 			<span class="year"> M2 </span> <br>
				 			<span class="tag"> Epigenomics </span>
				 			<span class="tag"> Pipeline </span>
				 			<span class="tag"> Jupyterhub </span>
				 			<span class="tag"> ChIP-seq </span>
				 			<span class="tag"> <i> E. Coli </i> </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	Pipeline for epigenomics data analysis (ChIP-seq experiment on <i>E. Coli</i>). <br><br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> <b> FASTQC </b> - Quality control of the reads and statistics. <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> <b> Bowtie </b> - Mapping. <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> <b> Picard MarkDuplicates </b> - Estimation of the number of duplicated reads. <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> <b> Deeptools </b> - ChIP quality control. <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> <b> IGV </b> - Data visualisation. <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> <b> MACS2 </b> - Peak calling. <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> <b> RSAT </b> - Motif Discovery. <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> <b> HOMER </b> - Peaks annotation. <br>
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	Hidden Markov Model and their application for gene prediction
				    	<div class="card__details">
				 			<span class="year"> M2 </span> <br>
				 			<span class="tag"> HMM </span>
				 			<span class="tag"> Modelling </span>
				 			<span class="tag"> Prediction </span>
				 			<span class="tag"> R </span>
				 			<span class="tag"> GenMarkS package </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	<b> Subject: </b> From a public bacterial genome's sequence, model this sequence using a Markov Model and use a Hidden Markov Model to predict coding regions. <br> <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Find a Markov Model fitted to the data using (1) Likelihood ratio test and (2) AIC and BIC criteria. <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Detect coding and non coding regions: <br> 
				    	(1) Learn a M1-M0 model. <br>
				    	(2) Use Viterbi algorithm to predict the most probable hidden state sequence. <br>
				    	(3) Repeat for a M1-M1 and a SM1-M1 model. <br>
				    	(4) Compare the results with a gene prediction tool (GeneMarkS).
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	R Shiny application development for functional enrichment analysis
				    	<div class="card__details">
				 			<span class="year"> M2 </span> <br>
				 			<span class="tag"> Functional annotation </span>
				 			<span class="tag"> Differential analysis </span>
				 			<span class="tag"> GSEA </span>
				 			<span class="tag"> ORA </span>
				 			<span class="tag"> Shiny application </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Assign <b>genes annotations</b> for a given species : Gene Ontology terms, Reactome and KEGG pathways, and Protein domains . <br> <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Perform <b>enrichment analysis</b> using 2 approaches : Gene Set Enrichment Analysis (GSEA) and Over Representation Analysis. <br> <br>
				    	Enrichment Analyses were performed using the <b>ClusterProfiler</b> R package and were manually re-implemented in R as well. We used <b>Pathview</b> to create figures of the pathways and <b>BiomartR</b> to convert gene ids. All figures were created using <b>ggplot2</b>.
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	Docking of donecopride in the human AChE model
				    	<div class="card__details">
				 			<span class="year"> M2 </span> <br>
				 			<span class="tag"> Structural biology </span>
				 			<span class="tag"> Molecular modeling </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Analysis of publicly available 3D complexes of human AChE co-cristallised with denepezil and galanthamine (PyMOL).<br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Docking of donepezil and donecopride in the AChE model with Vina Autodock.
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	Scripting project: Automatic annotation Datatable
				    	<div class="card__details">
				 			<span class="year"> M1 </span> <br>
				 			<span class="tag"> BioPython </span>
				 			<span class="tag"> tkinter </span>
				 			<span class="tag"> Public databases </span>
				 			<span class="tag"> DataTables </span>
				 			<span class="tag"> GitHub </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Collection of scripts to <b>aggregate gene annotations</b> in an interactive tabulated file for a given species. We used BioPython, the REST API and the NCBI E-utilities API to retrieve annotations from several public databases (NCBI, Ensembl, Uniprot, Prosite, Reactome, KEGG, Gene Ontology). The interactive table was created with DataTables. <br> <br> 
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Development of a <b>User Interface</b> with the tkinter Python library.
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	Simulation of a neural network with Neurons
				    	<div class="card__details">
				 			<span class="year"> L3 </span> <br>
				 			<span class="tag"> Neurons </span>
				 			<span class="tag"> Neurosciences </span>
				 			<span class="tag"> Simulation </span>
				 			<span class="tag"> Neural Network </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	<b> Subject: </b> Create a neural network of subthalamic nucleus neurons in <i>Rattus norvegicus</i> and study this network's behaviour by simulation. <br> <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Creation of 4 independent subthalamic nucleus neurons. <br> 
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Creation of dendritic trees <br>
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Construction of a neuronal connexion. <br> 
				    	<i class="fa fa-angle-double-right" aria-hidden="true"></i> Simulation and observation of the somatic membrane potential of different neurons. Analysis of the differences between the signals.
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	Experimental modelling of flute and guitar sounds
				    	<div class="card__details">
				 			<span class="year"> hobby </span> <br>
				 			<span class="tag"> R </span>
				 			<span class="tag"> Audacity </span>
				 			<span class="tag"> Signal processing </span>
				 		</div>
					</div>
				    <div class="card__face card__face--back">
				    	<b> Analysis of the signals characteristics in the switch from a right to a wrong key. </b> <br> <br> 
				    	Sounds were gathered with audacity and saved in R as Wave objects (tuneR package). <br> 
				    	Several parameters were observed : signal amplitude during time, harmonics and spectral analysis, autocorrelation of time-based variables. <br>
				    	Finally, I observed what happened during the transition from a right to a wrong key.
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	Creation of a database with PostgreSQL
				    	<div class="card__details">
				 			<span class="year">L3</span> <br>
				 			<span class="tag"> PostgreSQL </span>
				 			<span class="tag"> RDBMS </span>
				 			<span class="tag"> Relational model </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	<b> Subject: </b> As an artists agent, you have to create a database to keep track of all your clients. <br> <br>
				    	We decided (in pair) to create a database of French rappers and regroupep informations such as the type of contracts, our artists commitmments, their salary, royalties, their agenda as well as the time, date, and location of their concerts, showcases and other events. <br> <br>
				    	The project was carried out in 4 steps : a - Database modelling; b - Creation and filling of the tables; c - Commom operations and mandatory requests; d - Other requests database specific. 
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				    <div class="card__face card__face--front">
				    	C implementation of the Tablut game
				    	<div class="card__details">
				 			<span class="year">L3</span> <br>
				 			<span class="tag"> C </span>
				 			<span class="tag"> Analytical skills</span>
				 			<span class="tag"> Algorithm </span>
				 		</div>
				    </div>
				    <div class="card__face card__face--back">
				    	Tablut is a traditional Viking Game. One player controls the Shields and the other controls the Swords. The Swords player must try to capture the Chief Shield, while the Shields player must clear a safe path for the Chief Shield to escape off the board. Pieces can move any number of spaces in a straight line, and are captured when bracketed on two opposite sides by opponent pieces. <br> <br> 
				    	<span class="link"> <a href="https://federicoterzi.com/webTablut/" target="_blank"><i class="fa fa-gamepad" aria-hidden="true"></i> Play tablut online</a> </span>
				    </div>
				</div>
			</div>
			<div class="scene scene--card">
				<div class="card">
				 	<div class="card__face card__face--front"> 
				 		Integration of a tool for functional annotation and of a module for transposable elements characterisation in the pipeline MicroAnnot
				 		<div class="card__details">
				 			<span class="year">DUT</span> <br>
				 			<span class="tag"> Microsporidia </span>
				 			<span class="tag"> Annotation </span>
				 			<span class="tag"> Perl </span>
				 			<span class="tag"> Transposable Element </span>
				 			<span class="tag"> InterProScan </span>
				 		</div>
				 	</div>
				    <div class="card__face card__face--back"> 
				    	Microsporidian genome sequencing highlighted the presence of <b>transposable elements</b> with a considerable diversity. MicroAnnot integrates a module based on the exploitation of TransposonPSI and RepeatMasker that allows the identification of genes. We integrated InterProScan to MicroAnnot to perform the <b>functional analysis</b> of these genes. The use of a specific microsporidian transposable elements database allowed an extensive characterization of the genes within the genome of <i>Tubulinosema ratisbonensis</i> as well as their functional annotation.
				    </div>
				</div>
			</div>
			
		</center>
		<script type="text/javascript">
			var cards = document.querySelectorAll('.card');
			[...cards].forEach((card)=>{
			  card.addEventListener( 'click', function() {
			    card.classList.toggle('is-flipped');
			  });
			});
		</script>
	</body>
</html>